STEERING-COMMITTEE.md

```diff
@@ -95,8 +95,8 @@ first name):
 | -------------------------------------------------------------- | ---------------- | ------------ | ---------------------------------------------------- | ---------- | --------
 | <img width="30px" src="https://github.com/thisisnotapril.png"> | April Kyle Nassi | Google       | [@thisisnotapril](https://github.com/thisisnotapril) | Bootstrap  | 2021     |
 | <img width="30px" src="https://github.com/mbehrendt.png">      | Michael Behrendt | IBM          | [@mbehrendt](https://github.com/mbehrendt)           | Bootstrap  | 2021     |
-| <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Independent  | [@pmorie](https://github.com/pmorie)                 | 2020-12-11 | 2022     |
-| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas      | Independent  | [@vaikas](https://github.com/vaikas)                 | 2020-12-11 | 2022     |
+| <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Apple        | [@pmorie](https://github.com/pmorie)                 | 2020-12-11 | 2022     |
```

Comment:
I'm not representing my employer, so I'd prefer to remain listed as independent. Thank you